docs/writing-tests/test-runner.md

```diff
@@ -0,0 +1,293 @@
+---
```

Comment:
At what point does it make sense to format the README to fit the structure we want for docs, and embed its content for this page?

Because this content seems like it's 90% the same.

Reply:
Actually we want to do the opposite. I spoke to @shilman about this. The idea is to move everything to docs and then we can have the Reamde point to the Storybook docs.

Reply:
That works too! 💯

Reply:
Marking this as solved.